third_party/2/enum.pyi

```diff
@@ -8,10 +8,11 @@ _S = TypeVar('_S', bound=Type[Enum])
 # This is a temporary workaround to allow multiple creation of enums with builtins
 # such as str as mixins, which due to the handling of ABCs of builtin types, cause
 # spurious inconsistent metaclass structure. See #1595.
-class EnumMeta(ABCMeta, Iterable[Enum], Sized, Reversible[Enum], Container[Enum]):
+# Structurally: Iterable[T], Reversible[T], Container[T] where T is the enum itself
+class EnumMeta(ABCMeta):
     def __iter__(self: Type[_T]) -> Iterator[_T]: ...
     def __reversed__(self: Type[_T]) -> Iterator[_T]: ...
-    def __contains__(self, member: Any) -> bool: ...
+    def __contains__(self: Type[_T], member: Any) -> bool: ...
```

Comment:
Why did you add this?

Reply:
For consistency, with the Python 3 stub. I can remove it everywhere, instead, i suppose.

Reply:
I suppose it doesn't do anything; no need to change the other stub now.